### PR TITLE
update-packages run

### PR DESCRIPTION
**Pull request by iskakaushik**

### CODE REVIEW

**Comment by zanderso** [comment]
@jonahwilliams Do you know what is up with the dwds failures?

**Comment by jonahwilliams** [comment]
It looks it failed to download:

```
Downloading dwds 0.3.3...
Downloading args 1.5.2...
Downloading archive 2.0.11...
Changed 98 dependencies!
packages/flutter_tools/lib/src/build_runner/resident_web_runner.dart:8:8: Error: Error when reading '/root/.pub-cache/hosted/pub.dartlang.org/dwds-0.3.3/lib/dwds.dart': No such file or directory
import 'package:dwds/dwds.dart';
```

Can you kick it and try again?

**Comment by iskakaushik** [comment]
@jonahwilliams re-kicking does not seem to do it.

**Comment by zanderso** [comment]
@iskakaushik what happens when you run the analyzer check locally?

File: packages/flutter_tools/pubspec.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ dependencies:
   # To update these, use "flutter update-packages --force-upgrade".
   archive: 2.0.11
   args: 1.5.2
-  dwds: 0.7.9
+  dwds: 0.3.3
```
Comment (shihaohong) [inline packages/flutter_tools/pubspec.yaml]: Somehow this is downgrading here. The [last change](https://github.com/flutter/flutter/pull/44344) was _upgrades_ the version.

**Comment by shihaohong** [comment]
I kept running into this problem on working on a separate issue whenever I ran `flutter update-packages --force-upgrade`

**Comment by jonahwilliams** [comment]
Oh, I didn't even notice that ... you might be able to work around the issue by pinning the version in update_packages.dart and rebuilding the tool

**Comment by zanderso** [comment]
> Oh, I didn't even notice that ... you might be able to work around the issue by pinning the version in update_packages.dart and rebuilding the tool

We should spend some time figuring out why it is downgrading instead of upgrading. Isn't there a command line flag to pub to get it to spit out its constraint graph?

**Comment by jonahwilliams** [comment]
Yes, `pub --verbosity=solver`

**Comment by iskakaushik** [comment]
Ok, I think the solution is to have a version of `dwds` that relies on `built_value` with version `^7.0.0`. Based on running it with `--verbosity=solver`. Full log is: https://gist.github.com/iskakaushik/a31635125797f3d22ffd05648b652d9a. I think the most relevant line is:

```
conflict: dwds >=0.4.0 depends on built_value ^6.7.0
```

We seem to bring in a bunch of package updates that enforce a new major version of `built_value` and my suspicion is that the older version of `dwds` is the only one that satisfies this constraint.

To get to this we ended up downgrading `dwds` and `webkit_inspection_protocol`. @jonahwilliams , @zanderso how do you recommend I proceed here?

**Comment by jonahwilliams** [comment]
It sounds like we need an updated version of dwds with a bumped built_value dependency cc @grouma @jakemac53

**Comment by grouma** [comment]
Oops. Thought I resolved this already. Looks like I only fixed the downstream but not `package:dwds` itself. https://github.com/dart-lang/webdev/pull/838

**Comment by iskakaushik** [comment]
@grouma , thanks for releasing a new version. I'm trying this again.

**Review by dnfield — APPROVED** [review]
LGTM

**Pull request merged**